ipfx/hbg_dataset.py

```diff
@@ -93,3 +86,15 @@ def get_stimulus_code_ext(self, sweep_num):
 
     def get_recording_date(self):
         return self.nwb_data.get_recording_date()
+
+def get_finite_or_none(d, key):
+
+    try:
+        value = d[key]
+    except KeyError:
```

Comment:
`d.get(key)` is imo a much more succinct and explicit way to do this- get() returns none by default, or you can specify a default value, e.g. `d.get(key, None)`. It is very slightly slower though, so if this is done a whole lot maybe just leave it.